src/client/components/app/Navbar/index.tsx

```diff
@@ -109,9 +117,22 @@ export const Navbar = ({
     Icon: getNetworkIcon(network),
   }));
 
+  const secondTitleEnabled = !!subTitle?.length;
```

Comment:
nit: I think we don't need to check for length
```suggestion
  const secondTitleEnabled = !!subTitle;
```

Reply:
true, I would just leave it for another pr for now

Reply:
> Huge PR, went through all the files and only spotted minor stuff.
> 
> Trusting that the styles are looking good, only reviewed the code itself.

reviewed at:


> feat: revamp UI [WEB-1381] (

reviewed at #536